src/dmd/dscope.d

```diff
@@ -66,6 +66,8 @@ enum SCOPE
     // The following are mutually exclusive
     printf        = 0x4_0000, /// printf-style function
     scanf         = 0x8_0000, /// scanf-style function
+
+    subfuncscope  = 0x10_0000, /// in a function and in a compound scope
```

Comment:
Is this really needed? The old code was able to unambiguously detect these cases without it.

Reply:
it seems it wasnt.